docs/docusaurus.config.ts

```diff
@@ -59,6 +62,11 @@ const config: Config = {
           position: 'left',
           label: 'Documentation',
         },
+        {
+          href: 'https://blog.re-cinq.com',
```

Comment:
I'm not actually finding this hyperlink on the site

Reply:
Top left:
![image](https://github.com/re-cinq/aether/assets/19777480/d708a359-e471-48fd-b3e6-250b00f307e2)